### ISSUE TITLE
problem adding Cloud Connect endpoint one when interface no longer has VLANs

### ISSUE THREAD

**Issue by ajragusa** [GlobalNOC/OESS#1233]
When a CloudConnect endpoint has an interface that no longer has VLANs available the entire interface is disabled for being added to the connection.  Once you make sure there is a VLAN tag available on it you can provision to any of the interfaces that had a VLAN tag available.  This probably happens for non-CloudConnect interfaces as well

**Comment by mattzekauskas** [comment]
More specifically, adding two new AWS interfaces I added an ACL that permitted the AWS Monitoring workgroup to a single VLAN on each interface.   I created the first circuit in the AWS Monitoring workgroup.   I went to create the second circuit (to a different VLAN on a different port) and while the expected VLAN popped up as allowed, the bandwidth was greyed out, and I wasn't allowed to add the endpoint (in the AWS Monitoring workgroup).    In fact, I couldn't allocate on any of the existing AWS connections, although they all had bandwidth.   When I widened the ACL to permit all VLANs on the first interface to AWS Monitoring (and left the second one with a single VLAN I had not yet allocated alone), the bandwidth was no longer greyed out, and I could now add the endpoint on the second interface.
